fluent-pseudo/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+use once_cell::sync::Lazy;
```

Comment:
@zbraniecki / @waywardmonkeys Any quick opinions about whether it makes sense given its position only in fluent-pseudo to introduce this dependency to get rid of the reliance on unsafe undefined behavior? If/when we bump the MSRV up a bit we could of course get rid of this again and use the implementation of this in *std*, but this way we can at least do a minor release without affecting our current MSRV while also not having to suppress clippy's objections.

Reply:
this looks good to me.